### ISSUE TITLE
The TDD assertion style 'assert' methods are not working

### ISSUE THREAD

**Issue by jennifer-shehane** [cypress-io/cypress#101]
[None of the methods](http://chaijs.com/api/assert/) for `assert` are available on `assert`.

**Example Code:**

``` javascript
it('asserts', function(){
  assert.isTrue(true, "true should be true")
})
```

**Error**
`assert.isTrue is not a function`

**Comment by brian-mann** [comment]
The code for this is done, but this has yet to be released. We'll update the issue and reference the changelog when it's released.

**Comment by brian-mann** [comment]
Fixed in [`0.20.0`](https://docs.cypress.io/guides/references/changelog.html#0-20-0)
